src/Compilers/Core/Portable/CommandLine/CommonCompiler.cs

```diff
@@ -788,6 +788,16 @@ private int RunCore(TextWriter consoleOutput, ErrorLogger errorLogger, Cancellat
             // We pass it to the generators, which will realize any symbols they require. 
             compilation = RunGenerators(compilation, Arguments.ParseOptions, generators, additionalTexts, diagnostics);
 
+            // https://github.com/dotnet/roslyn/issues/44087 
+            // Workaround by re-getting options for *all* the syntax trees including generated ones if there were
```

Comment:
Is the `all` portion of this statement still true?

Reply:
Derp. Fixed.